### ISSUE TITLE
Document another example for the internal registry access

### ISSUE THREAD

**Issue by gbraad** [code-ready/crc#3215]
In #48 we introduced semi-supported documentation about accessing the internal registry. We detail to use an example image, which does not use a 'standard' image. We could consider an alternative image.

**Comment by gbraad** [comment]
@praveenkumar What would you suggest? Since this is actually not part of the main documentation flow to get you started, it might be OK to use the UBI images. Do take note, we have to avoid to overcomplicate this part of the documentation as it should be a solution instead of elaborating as much knowledge as possible.

**Comment by praveenkumar** [comment]
@gbraad https://hub.docker.com/r/openshift/hello-openshift/ not a good link to pick because it is outdated, I am OK with https://access.redhat.com/documentation/ru-ru/red_hat_container_development_kit/2.1/html/getting_started_guide/deploying_an_application_on_openshift steps. Also we can replace https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.3/html/getting_started_guide/using_gsg#accessing-the-internal-openshift-registry_gsg `podman pull quay.io/libpod/alpine` part to `podman pull <registry>/<image>` and says (i.e quay.io/libpod/alpine) one.

**Comment by gbraad** [comment]
It is VERY outdated (3 years!), but it is more about the idea than the actual image.

**Comment by robin-owen** [comment]
I agree. IMHO, an example using either Fedora or CentOS as a base would be ideal if a UBI example is unrealistic.

**Comment by gbraad** [comment]
> unrealistic

this is due to a missing functionality in the podman-remote clients. Not sure what this is about, .... (waiting for an issue reference).

**Comment by cfergeau** [comment]
The podman issue is in the bug linked in the initial description https://bugzilla.redhat.com/show_bug.cgi?id=2093012#c6

**Comment by gbraad** [comment]
@cfergeau I suggested to file separate issues to track this better as the reporter also concludes: https://bugzilla.redhat.com/show_bug.cgi?id=2093012#c7 the issue talks about 3 different things. This is vey convoluted to follow and handle in that case. For the documentation side, this was created.

Who will otherwise follow up on ` --remove-signatures` for podman?
